lib/propre.rb

```diff
@@ -39,11 +39,9 @@ def self.remove_patterns(arg)
   end
 
   def self.sanitize(arg)
-    if arg.count('.') > arg.count('-')
-      arg.gsub('.', ' ').strip
-    else
-      arg.gsub('-', ' ').strip
-    end
+    separator = %w(- _ .).each_with_object({}) { |c, s| s[c] = arg.count(c) }
+      .sort_by(&:last).reverse.first.first
+    arg.gsub(separator, ' ').strip
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.